src/conversions.jl

```diff
@@ -90,31 +155,31 @@ function astimezone(zdt::ZonedDateTime, tz::FixedTimeZone)
 end
 
 function zdt2julian(zdt::ZonedDateTime)
-    datetime2julian(utc(zdt))
+    datetime2julian(DateTime(zdt, UTC))
 end
 
 function zdt2julian(::Type{T}, zdt::ZonedDateTime) where T<:Integer
-    floor(T, datetime2julian(utc(zdt)))
+    floor(T, datetime2julian(DateTime(zdt, UTC)))
 end
 
 function zdt2julian(::Type{T}, zdt::ZonedDateTime) where T<:Real
-    convert(T, datetime2julian(utc(zdt)))
+    convert(T, datetime2julian(DateTime(zdt, UTC)))
 end
 
 function julian2zdt(jd::Real)
     ZonedDateTime(julian2datetime(jd), utc_tz, from_utc=true)
 end
 
 function zdt2unix(zdt::ZonedDateTime)
-    datetime2unix(utc(zdt))
+    datetime2unix(DateTime(zdt, UTC))
 end
 
 function zdt2unix(::Type{T}, zdt::ZonedDateTime) where T<:Integer
-    floor(T, datetime2unix(utc(zdt)))
+    floor(T, datetime2unix(DateTime(zdt, UTC)))
 end
 
 function zdt2unix(::Type{T}, zdt::ZonedDateTime) where T<:Real
-    convert(T, datetime2unix(utc(zdt)))
+    convert(T, datetime2unix(DateTime(zdt, UTC)))
```

Comment:
Let's keep the addition of `return` statements to a separate PR otherwise this PR could become held up.

Reply:
https://github.com/invenia/BlueStyle/issues/43

Reply:
I don't see why clicking "apply" would hold this up.
No need to go do the others I've missed but might as well get these ones.

Reply:
What I mean is I'm good with these changes without the applying the `return` statements

Reply:
what I mean is:
Rather than having this discussion, you can click "Commit suggestion" above.
And that will take less time than concluding the discussion.

Reply:
Transfering from slack:

@omus  said:
> In the case of a disagreement follow the status quo of what is currently present in the repo 

I said:
> That is an important distinction that I failed to take into account.
> I am now convinced.